osutil/kcmdline/kcmdline.go

```diff
@@ -441,3 +445,39 @@ func (kap *ArgumentPattern) UnmarshalYAML(unmarshal func(interface{}) error) err
 
 	return nil
 }
+
+func (kap *ArgumentPattern) marshalToString() (string, error) {
+	return fmt.Sprintf(`%s=%s`, kap.param, kap.value.String()), nil
```

Comment:
The error can be removed from the return type
```suggestion
func (kap *ArgumentPattern) marshalToString() string {
	return fmt.Sprintf(`%s=%s`, kap.param, kap.value.String())
```

Reply:
I should have re-read my code. I had something else in mind at first. Will fix that.